include/aos/common/tools/string.hpp

```diff
@@ -496,7 +496,7 @@ class String : public Array<char> {
         Clear();
 
         // cppcheck-suppress wrongPrintfScanfArgNum
-        auto ret = snprintf(Get(), MaxSize() + 1, format, args...);
```

Comment:
Is there any warning here? Should be done in separate commit with proper commit message.